src/libraries/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -2267,6 +2267,7 @@ public void TestLongProcessIsWorking()
         [Fact]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/52852", TestPlatforms.MacCatalyst)]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/53095", TestPlatforms.Android)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/29383", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
```

Comment:
I would disable it on OSX for all runtime flavors. If it is failing on NativeAOT OSX, it is going to flaky outside NativeAOT OSX too.

On the other hand, I believe that the test is not flaky anymore on Linux, so we can keep it enabled there.

Note that the test used to be disabled on OSX up until my recent change https://github.com/dotnet/runtime/pull/111850/files#diff-f756f37255fb9daf54bb56c6b6653e2dd6b5acb1103d60844ab08433e8053fe5R2266 . I have enabled since it seems to be passing, but it looks like that it was premature.

Reply:
Sounds good!

```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/29383", TestPlatforms.OSX)]
```